pages/common/jest.md

```diff
@@ -0,0 +1,23 @@
+# jest
+
+> Delightful JavaScript Testing.
+
+- Run all tests:
+
+    jest
+
+- Run tests filtered by spec name:
+
+    jest -t name-of-spec
```

Comment:
spec_name

Reply:
There's no precedence for this formatting in the [Jest CLI Options](https://jestjs.io/docs/en/cli) docs. The `jest --help` version however is probably more precise:

> Run only tests with a name that matches the regex pattern.

Reply:
Sorry, I don't understand. You wrote "name-of-spec". Since our pages follow the snake case convention, I suggested spec_name, which also matches tokens like package_name in other pages.

If you change the description to regex, you can change the token to a concrete value like "booking*"

Reply:
Ah right, I misunderstood. Will change for the command example 👍